crypto/hasher/coniks/coniks.go

```diff
@@ -0,0 +1,75 @@
+// TODO(huyvq): Remove package import.
+// Other package shouldn't need to import this package,
+// instead they should insert `hasher` and import this
+// package using a blank import name.
+// Should be adressed in #06.
+
+package coniks
+
+import (
+	"crypto"
+
+	"github.com/coniks-sys/coniks-go/crypto/hasher"
+	"github.com/coniks-sys/coniks-go/utils"
+)
+
+func init() {
+	hasher.RegisterHasher(CONIKSHasher, New)
+}
+
+const (
+	// CONIKSHasher is the identity of the hashing algorithm
+	// specified in the CONIKSHasher paper.
+	CONIKSHasher = "CONIKS Hasher"
```

Comment:
I wonder, if there will be different hashers, shouldn't be a central place of identifiers? I always thought we are only swapping the underlying hash function, but all these hashers will be "CONIKS Hashers".

Reply:
The idea is each hasher implementation should have its own ID declared in its own package. But you're right, we should have a central place of identifiers which includes our "standard" hash schemes, and "CONIKS Hasher" should be replaced by something like "SHA256-xxxx".